packages/headless/src/app/search-engine/search-engine.ssr.test.ts

```diff
@@ -40,5 +59,46 @@ describe('SSR', () => {
       const resultList = buildResultList(engine);
       expect(resultList.state.results.length).toBe(10);
     });
+
+    it('from build result, hydrates engine and fetches results using hydrated engine', async () => {
```

Comment:
```suggestion
    it('hydrates engine and fetches results using hydrated engine from build result', async () => {
```

should read natural with starting with 'it' - 'it hydrates ..'